code/datums/wounds/cuts.dm

```diff
@@ -0,0 +1,259 @@
+	var/demotes_to
+
+	/// How much staunching per type (cautery, suturing, bandaging) you can have before that type is no longer effective for this cut NOT IMPLEMENTED
+	var/max_per_type
+	/// The maximum flow we've had so far
+	var/highest_flow
+	/// How much flow we've already cauterized
+	var/cauterized
+	/// How much flow we've already sutured
+	var/sutured
+
+	/// The current bandage we have for this wound (maybe move bandages to the limb?)
+	var/obj/item/stack/current_bandage
+	/// A bad system I'm using to track the worst scar we earned (since we can demote, we want the biggest our wound has been, not what it was when it was cured (probably moderate))
+	var/datum/scar/highest_scar
+
+/datum/wound/brute/cut/apply_wound(obj/item/bodypart/L, silent = FALSE, datum/wound/brute/cut/old_wound = null)
+	. = ..()
+	blood_flow = initial_flow
+	if(old_wound)
+		blood_flow = max(old_wound.blood_flow, initial_flow)
+		if(old_wound.severity > severity && old_wound.highest_scar)
+			highest_scar = old_wound.highest_scar
+			old_wound.highest_scar = null
+		if(old_wound.current_bandage)
+			current_bandage = old_wound.current_bandage
+			old_wound.current_bandage = null
+
+	if(!highest_scar)
+		highest_scar = new
+		highest_scar.generate(L, src, add_to_scars=FALSE)
+
+/datum/wound/brute/cut/remove_wound(ignore_limb, replaced)
+	if(!replaced && highest_scar)
+		already_scarred = TRUE
+		highest_scar.lazy_attach(limb)
+	return ..()
+
+/datum/wound/brute/cut/get_examine_description(mob/user)
+	if(!current_bandage)
+		return ..()
+
+	var/bandage_condition = ""
+	// how much life we have left in these bandages
+	switch(current_bandage.absorption_capacity)
+		if(0 to 1.25)
+			bandage_condition = "nearly ruined "
+		if(1.25 to 2.75)
+			bandage_condition = "badly worn "
+		if(2.75 to 4)
+			bandage_condition = "slightly bloodied "
+		if(4 to INFINITY)
+			bandage_condition = "clean "
+	return "<B>The cuts on [victim.p_their()] [limb.name] are wrapped with [bandage_condition] [current_bandage.name]!</B>"
+
+/datum/wound/brute/cut/receive_damage(wounding_type, wounding_dmg, wound_bonus)
+	if(wounding_type == WOUND_SHARP)
+		blood_flow += 0.05 * wounding_dmg
+
+/datum/wound/brute/cut/handle_process()
+	if(victim.reagents && victim.reagents.has_reagent(/datum/reagent/medicine/coagulent))
+		blood_flow -= 0.25
+	if(current_bandage)
+		if(clot_rate > 0)
+			blood_flow -= clot_rate
+		blood_flow -= current_bandage.absorption_rate
+		current_bandage.absorption_capacity -= current_bandage.absorption_rate
+		if(current_bandage.absorption_capacity < 0)
+			victim.visible_message("<span class='danger'>Blood soaks through \the [current_bandage] on [victim]'s [limb.name].</span>", "<span class='warning'>Blood soaks through \the [current_bandage] on your [limb.name].</span>", vision_distance=COMBAT_MESSAGE_RANGE)
+			QDEL_NULL(current_bandage)
+			treat_priority = TRUE
+	else
+		blood_flow -= clot_rate
```

Comment:
why haven't you added the clot_rate>0 check here as well?

Reply:
Because if the clot rate is negative and making the wound worse, which it is for critical, I want to ignore that opening while bandaged